### ISSUE TITLE
1.18.2 crashing on startup

### ISSUE THREAD

**Issue by UltraBlackLinux** [maruohon/tweakeroo#334]
Hey there,
It seems like tweakeroo is not compatible with Minecraft 1.18.2 yet, it throws an error and crashes:
```
[02/03/2022 21:25:23 pm] [Render thread/ERROR] Mixin apply for mod tweakeroo failed mixins.tweakeroo.json:MixinMinecraftClient -> net.minecraft.class_310: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException @Shadow method method_1536 in mixins.tweakeroo.json:MixinMinecraftClient was not located in the target class net.minecraft.class_310. Using refmap tweakeroo-fabric-1.18.1-refmap.json
...
[02/03/2022 21:25:23 pm] Caused by: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException: @Shadow method method_1536 in mixins.tweakeroo.json:MixinMinecraftClient was not located in the target class net.minecraft.class_310. Using refmap tweakeroo-fabric-1.18.1-refmap.json
```
I only had LazyDFU, MaliLib, MixinTrace, NEC and Tweakeroo enabled and was able to reproduce it reliably.
(I sure hope this is not some weirdass issue on my side again)
Is this being worked on already, or did it just go unnoticed?
Thanks!

**Comment by Zombiedenden** [comment]
I'm having the same problem

**Comment by Crec0** [comment]
mod was not updated to 1.18.2
ofc it wont work and will likely crash considering the changes to 1.18.2 were not small

**Comment by UltraBlackLinux** [comment]
> mod was not updated to 1.18.2 ofc it wont work and will likely crash considering the changes to 1.18.2 were not small

Yeah it sure does make sense, but considering the time that has passed since the update and the apparent inactivity of this repo (this sounds rude and impatient, but that wasn't my intention), I would have guessed that it's just supposed to work with 1.18.2

**Comment by maruohon** [comment]
The reason it "still hasn't updated" is because I honestly don't feel like working on these pointless MC version updates all the time. I'm currently mostly focusing on the mod rewrites in 1.12.2, working on cleaning u p the code base of all the mods, and thus working on code that I don't actively hate every moment I have to stare at it, like I do in the old code base that is still in all the 1.13+ versions of the mods.

All the time I have to waste porting stuff to new minor releases just further delays these rewrites and makes it more likely that they will never see the light of day in 1.13+. And being forced to work on the 1.17+ stuff usually just completely destroys my modding motivation and I end up just procrastinating watching youtube or something, or just sleeping though most of the day to avoid working on it.

So basically I'll port them at some point soon, when I feel like it. Earlier today I worked on a bunch of bug fixes for MiniHUD for 1.18.1, and then I just got burnt out and had to switch back to 1.12.2 stuff that I'm actually excited to work on. I still need to finish one bug fix for MiniHUD, and also probably try to improve the omega crappy `massCraft` code in Item Scroller, to get the "final" 1.18.1 updates out before I move everything to 1.18.2.

**Comment by Zombiedenden** [comment]
@maruohon 
Well, whether you make it or not, I hope you know that you are amazing and your work is really appreciated. It's really outstanding. :)

**Comment by UltraBlackLinux** [comment]
Yep, I agree. Many mod devs just stop updating their mods for older versions, which I expected to happen here as well. 
Thank you for your dedication, and it's all free!

**Comment by UltraBlackLinux** [comment]
@maruohon Well I am impressed! This went quicker than I expected after your previous reply, thanks!
